test/contracts/consumer/fabric8auth/consumer_test.go

```diff
@@ -0,0 +1,69 @@
+package fabric8auth
+
+import (
+	"log"
+	"os"
+	"testing"
+
+	"github.com/fabric8-services/fabric8-auth/test/contracts/model"
+	"github.com/fabric8-services/fabric8-wit/test/contracts"
+	"github.com/pact-foundation/pact-go/dsl"
+	"github.com/pact-foundation/pact-go/types"
+)
+
+// TestWitAPIConsumer runs all user related tests
+func TestWitAPIConsumer(t *testing.T) {
+	log.SetOutput(os.Stdout)
+
+	var pactDir = os.Getenv("PACT_DIR")
+	var pactVersion = os.Getenv("PACT_VERSION")
+
+	var pactConsumer = "Fabric8Wit"
```

Comment:
Please use the github names `fabric8-wit` and `fabric8-auth` and don't invent new spellings, okay?

Reply:
Thanks for the tip - fixed.